src/Faker/Provider/it_IT/Payment.php

```diff
@@ -16,4 +16,58 @@ public static function bankAccountNumber($prefix = '', $countryCode = 'IT', $len
     {
         return static::iban($countryCode, $prefix, $length);
     }
+
+    /**
+     * Source:
+     * @see http://www.comuni-italiani.it/banche/principali.html
```

Comment:
Perhaps we should use data from https://en.wikipedia.org/wiki/List_of_banks_in_Italy instead?

Reply:
indeed, the page linked by @xfudox has a copyright notice, we can't reuse the content in Faker, which uses an MIT license.

Reply:
ok, i'll correct this asap